diff -uraN vkQuake-1.31.0/Quake/mem.c omv-vkQuake-1.31.0/Quake/mem.c
--- vkQuake-1.31.0/Quake/mem.c	2024-06-22 11:53:48.000000000 +0200
+++ omv-vkQuake-1.31.0/Quake/mem.c	2024-06-22 17:02:59.195677175 +0200
@@ -35,7 +35,7 @@
 #if defined(__GNUC__) && (__GNUC__ >= 12)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wstringop-overflow" +
-#pragma message "ignore stringop-overflow warnings for mimalloc v2.14+"
+#pragma message "ignore stringop-overflow warnings for mimalloc v2.14"
 #endif
 
 #undef snprintf
