diff -Naur vkQuake-1.30.1/Quake/Makefile vkQuake-1.30.1.tpg/Quake/Makefile
--- vkQuake-1.30.1/Quake/Makefile	2023-03-15 02:59:38.000000000 +0000
+++ vkQuake-1.30.1.tpg/Quake/Makefile	2023-03-18 13:55:36.635545656 +0000
@@ -10,15 +10,15 @@
 
 ### Enable/Disable codecs for streaming music support
 USE_CODEC_WAVE = 1
-USE_CODEC_FLAC = 0
+USE_CODEC_FLAC = 1
 USE_CODEC_MP3 = 1
 USE_CODEC_VORBIS = 1
-USE_CODEC_OPUS = 0
+USE_CODEC_OPUS = 1
 # either xmp or mikmod (or modplug)
 USE_CODEC_MIKMOD = 0
 USE_CODEC_XMP = 0
-USE_CODEC_MODPLUG = 0
-USE_CODEC_UMX = 0
+USE_CODEC_MODPLUG = 1
+USE_CODEC_UMX = 1
 
 # which library to use for mp3 decoding: mad or mpg123
 MP3LIB = mad
